handlers/main.yml

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: MIT
+---
+# handlers file for nbde_client.
+- name: nbde_client update initramfs
+  when:
+    - __nbde_client_update_initramfs | bool
+    - __nbde_client_initramfs_update_cmd is defined
+    - __nbde_client_initramfs_update_cmd != ""
```

Comment:
you can combine the last two conditions as `__nbde_client_initramfs_update_cmd | d("")` - `""` should evaluate to `False` in this (boolean) context